webapp/src/components/onboarding/EditorTour.tsx

```diff
@@ -13,6 +13,8 @@ const Toolbox: TargetContent = {
     title: lf("Toolbox"),
     description: lf("Drag out blocks of code from the Toolbox categories into the Workspace."),
     targetQuery: "#blocksEditorToolbox",
+    sansQuery: "#projectNameArea",
```

Comment:
Code seems good but would it be easier in this case to just move the targetQuery here to `.blocklyToolboxDiv`? I would see the sansquery as more of a last resort of possible (e.g. in the other case the toolbox gets counted as part of the blocksEditor & the way the workspace svg is set up there's not an immediately obviously delineation on what the workspace is, but most of the time when you have an element that is 'too big' it's because there is a parent element constraining it / with `overflow: auto` on it)

Reply:
Yea this works. I think originally I selected the other query because it doesn't include the extra whitespace in the toolbox when it isn't expanded. But it is simpler this way, so I made the change. Thanks for the suggestion!